index.html

```diff
@@ -15,12 +15,21 @@
 
         <nav class="nav-bar">
           <a href="#" class="Left-Logo">My Logo</a>
-          <a> <img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"> </a>
+          <a> <img src="Images/Union.png" id="Hamburger-icon" alt="Three-Lines"> </a>
```

Comment:
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu should appears. For now nothing happens. kindly fix this.
![ha](https://user-images.githubusercontent.com/65860678/188519190-5e751f1f-77c1-43c5-a81b-1c94b3f91424.PNG)